R/pkg/R/DataFrame.R

```diff
@@ -1187,6 +1187,65 @@ setMethod("dapply",
             dataFrame(sdf)
           })
 
+#' gapply
+#'
+#' Apply a function to each group of a DataFrame.
+#'
+#' @param x A SparkDataFrame
+#' @param func A function to be applied to each group partition specified by grouping
+#'        column of the SparkDataFrame.
+#'        The output of func is a data.frame.
+#' @param schema The schema of the resulting DataFrame after the function is applied.
```

Comment:
data.frame - a local R data.frame, defined in R base packages
SparkDataFrame - represents spark's DataFrame in R
SparkDataFrame was previously called DataFrame. So it can be renamed to SparkDataFrame.